openid4vc-high-assurance-interoperability-profile-1_0.md

```diff
@@ -230,6 +230,8 @@ The following requirements apply to all OpenID4VP flows when the mdoc Credential
 * The ISO mdoc Credential Format specific DCQL parameter, `intent_to_retain` defined in Annex B.3.1 of [@!OIDF.OID4VP] MUST be present.
 * When multiple ISO mdocs are being returned, each ISO mdoc MUST be returned in a separate `DeviceResponse` (as defined in 8.3.2.1.2.2 of [@!ISO.18013-5]), each matching to a respective DCQL query. Therefore, the resulting `vp_token` contains multiple `DeviceResponse` instances.
 
+Note: The second edition of ISO/IEC 18013-5 ([@!ISO.18013-5.second.edition]) contains definitions of status / revocation mechanisms that can be used with ISO mdocs.
```

Comment:
Maybe this aligns with the discussion in https://github.com/openid/OpenID4VC-HAIP/pull/289#issuecomment-3361860575 : 
```suggestion
* The Credential Issuer MAY include Status / Revocation in the issued mdoc. When doing so, it MUST used one of the status / revocation mechanisms defined in ISO/IEC 18013-5 ([@!ISO.18013-5.second.edition]).
```